Sources/ftp/ftpConnection.cpp

```diff
@@ -3,7 +3,7 @@
 #include <string>
 #include <algorithm>
 #include <map>
-#include "outputLine.h"
+#include "../debug/outputLine.h"
```

Comment:
I'd very much prefer if we used paths relative to `Includes` (where I feel like "public" headers should live) over these paths starting with `..`.